### ISSUE TITLE
A11y_AzureDataStudio_Dashboard_Home-Toolbar_ScreenReader : Incorrect name as 'In collapse' is announced for a "^" control for screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/azuredatastudio#11551]
**"[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”**
GitHubTags:#A11y_AzureDataStudioJuly2020;#A11y_July27_2020_TestPass;#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#MAC;#Screenreader;#VoiceOver;#A11ySev2;#Benchmark;#MAS1.3.1;#MAS4.1.2;#MAS4.2.1;#FTP;

### Environment Details:
Application Name: Azure Data Studio
Application Version: 1.21.0-insider
Commit: eccf3cf
Date: 2020-07-24T09:28:31.172Z
VS Code: 1.48.0
Electron: 9.1.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina (Version 10.15.6 (19G73)
Screen Reader: VoiceOver
MAS References: MAS1.3.1, MAS4.1.2, MAS4.2.1

### Repro Steps:
1. Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Double click on connected server or right click on it & select manage option to open the Dashboard.
4. Navigate to Home under Dashboard & hit enter.
5. Start screen reader, the navigate to "^" control and listen to the announcement been made for this control.

### Actual:
When screen reader users navigate to the "^" control, it name is announced as 'Collapsed selected' which is incorrect. 

### Expected:
The "^" control should be provided with name as 'Show Details" with expand/collapse state for screen reader users so that users are able to identify it's state on interacting with it.

### User Impact:
If proper name and state of the control is not announced to the scree reader users the they will not understand how to interact with that control.

### Attachment link for Reference:
[11551_A11y_AzureDataStudio_Dashboard_Home-Toolbar_ScreenReader Incorrect name as 'In collapse' is announced for a "^" control for screen reader users.zip](https://github.com/microsoft/azuredatastudio/files/4986396/11551_A11y_AzureDataStudio_Dashboard_Home-Toolbar_ScreenReader.Incorrect.name.as.In.collapse.is.announced.for.a.Show.more.less.control.for.screen.reader.users.zip)

**Comment by manaswita-chougule** [comment]
@alanrenmsft 
Downgrading the severity to sev3.
Verified the issue in below environment:
SQL Azure Data Studio Version: 1.42.0-insider (Universal)
MAC Version: macOS Ventura 13.3

The issue is repro'ing, name should be provided as "Server Properties expand/collapse" button instead of "expand/collapse widget" button. Please find attachment for reference. Thanks!

![11551_Repro](https://user-images.githubusercontent.com/91586101/224293267-8e5627ec-5b09-4cb7-b237-e28bda67ca57.gif)
